include/common_types.hpp

```diff
@@ -15,6 +15,7 @@ At runtime just dlopen the dll, that's it!
 namespace chainblocks {
 
 enum class BasicTypes {
+  None,
   Any,
```

Comment:
Wondering if `Any` should have the value `-1` since it is a special case. Bonus point, only the value for `Any` would change from previously `0` to now `-1`. The numerical value for any other enum value would stay the same as before.

I'm saying that in case it causes a breaking change.

Reply:
It does not really matter honestly.
This enum is used only in Sequence, Table and now Substrate.Decode - the order and value does not matter for those and does not matter for mal either as they get only texual version.